ajax/libs/cachep2p/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "cachep2p",
+  "filename": "cachep2p.min.js",
+  "description": "The Peer to Peer Cache for the Masses",
+  "version": "0.1.5",
+  "author": {
+    "name": "Carlos Guerrero",
+    "email": "carlos@carlosguerrero.com",
+    "url": "https://cachep2p.com"
+  },
+  "homepage": "https://cachep2p.com",
```

Comment:
This site looks down, seems `http://www.cachep2p.com/` is the working one, can you confirm that? Maybe we should create an issue at upstream repo if you have the same problem.